### ISSUE TITLE
False positive unused import while formatting conditional

### ISSUE THREAD

**Issue by axemorgan** [pinterest/ktlint#886]
## Expected Behavior
Formatting this code should not remove the import for `CONSTANT`.

```kotlin
import other.CONSTANT

fun main() {
	if (CONSTANT 
    	&& true
    ) {
        return
    }
}
```

## Observed Behavior
Ktlint outputs the following (which no longer compiles due to `error: unresolved reference: CONSTANT`):
```kotlin
fun main() {
	if (CONSTANT &&
    	true
    ) {
        return
    }
}
```
## Steps to Reproduce
I created a sample to reproduce it [here](https://github.com/axemorgan/ktlint-issue). Format and then compile to reproduce the error.

Notes:
It seems like correcting the `&&` wrapping is key here, because other formatting fixes don't seem to trigger this same behavior.
Also if `CONSTANT` is referenced elsewhere in the same file, it will _not_ be removed during formatting.

## Your Environment
ktlint 0.37.2

**Comment by romtsn** [comment]
Could you try the latest version (0.38.1) and also run with --verbose to get an idea which rule is it?

**Comment by axemorgan** [comment]
I upgraded to 0.38.1 and it still happens. Verbose mode says this is the rule: `Line must not begin with "&&" (chain-wrapping)`

**Comment by paul-dingemans** [comment]
Problem with unused import was resolved in 0.38.1. The violation `Line must not begin with "&&" (chain-wrapping)` can simply be resolved by the user.
